propostas/proposal-2022-2.md

```diff
@@ -0,0 +1,72 @@
+# Decred Brazil - Communication and Content Production
+
+# What
+
+This proposal seeks funding for the reactivation and production of content for Decred Brazil social Media accounts such as Twitter, Instagram and creation of a brand new TikTok account.
+
+# Why
+
+We seek to continue and improve the production of Social Media content for Brazilian audience. In that way we can engage the community and spread our word to possible new investors and contributors.
+
+Brazilians are very addicted to internet and social media. They spend an average of 3 hours and 42 minutes per day on Social Media, according to Sortlist (marketing enterprise). In that way we can understand that actions in this segment represent an opportunity for Decred awareness growth.
```

Comment:
It would be nice to mention how addicted to crypto Brazilians are too. Example: https://www.coindesk.com/business/2021/10/18/brazilians-have-acquired-4b-in-cryptocurrencies-in-2021-central-bank-says/ and https://triple-a.io/crypto-ownership-brazil/. Good thing: people already know crypto. Bad thing: Brazilians still fall for crypto-based ponzi schemes (https://www.businesstoday.in/crypto/story/bitcoin-ponzi-schemes-wreak-havoc-on-brazils-new-egypt-320019-2022-01-23). High quality crypto education is very important and DCR can be a leader on this. Definitely not an easy task with all the ngu easy money scams around.